examples/realtime.rs

```diff
@@ -18,19 +18,13 @@ fn main() {
         sample_rate,
         buffer_size: cpal::BufferSize::Default,
     };
-    let boop = oddio::Frames::from_iter(
-        sample_rate.0,
-        // Generate a simple sine wave
-        (0..sample_rate.0 * DURATION_SECS).map(|i| {
-            let t = i as f32 / sample_rate.0 as f32;
-            (t * 500.0 * 2.0 * std::f32::consts::PI).sin() * 80.0
-        }),
-    );
-
-    let speed = 50.0;
 
+    // create our oddio handles for a `SpatialScene`. We could also use a `Mixer`,
+    // which doesn't have Spatialized audio in it.
     let (mut scene_handle, scene) = oddio::split(oddio::SpatialScene::new(sample_rate.0, 0.1));
 
+    // We send `scene` into this closure, where changes to `scene_handle` are reflected.
+    // from here on out, `scene_handle` is how we play audio and apply filters to it.
```

Comment:
```suggestion
    // `scene_handle` is how we add new sounds and modify the scene live.
```

`scene` doesn't actually expose much in the way of interesting controls to begin with, and we can't apply filters after the fact.